etest/build.py

```diff
@@ -1,32 +1,105 @@
 """Build command for etest Docker images."""
-import os
-import sys
+from enum import Enum
 from pathlib import Path
-from typing import List
 
 import click
 
+from etest import docker, qemu
+from etest.profile import Profile
+
+
+class __commands(Enum):
```

Comment:
I'd probably put these into etest.build._libc which will make it easier to find and indicate that the whole module is private.  Up to you though since this module isn't too unreasonable yet.